public/adminer.php

```diff
@@ -0,0 +1,2044 @@
+<?php
```

Comment:
install it on your machine, not in the project

Reply:
Oops sorry i forgot to delete adminer.